README.md

```diff
@@ -59,13 +59,21 @@ CmdStan path set to: {a path on your file system}
 
 If you run into trouble, consult the official [`cmdstanr`](https://mc-stan.org/cmdstanr/index.html) website for further installation guides and help.
 
-## Download this repository and install the project package (`wwinference`)
+## Install `wwinference`
+
 Once `cmdstanr` and `CmdStan` are installed, the next step is to download this repository and install the package, `wwinference`. The package provides tools for specifying and running the model, and installs other needed dependencies.
 
-Once you have downloaded this repository, navigate to it within an R session and run the following:
+### User install
+
+The package can be installed directly from github by running the following within an R session:
+```R
+install.packages("remotes")
+remotes::install_github("CDCgov/ww-inference-model")
+```
+### Developer install
+If you are interested in contributing to the `wwinference`, clone the repository, navigate to it within an R session, and run the following:
 
 ```R
-install.packages('remotes')
 remotes::install_local()
 ```
```

Comment:
I'd omit this from the main README. There are many ways to work with a development R package, and I don't think we need to prescribe one. I'm also concerned this will convince people who might like to contribute in the future that they should download and `install_local()` versus installing from Github, which is not what we mean.